### ISSUE TITLE
tutorial back, next buttons not highlighted when selected

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2301]
**Describe the bug**

there is no indication the back and next buttons are selected when tabbed to in a tutorial (they're not highlighted or anything like other buttons)

**To Reproduce**
1. Go to tutorial
2. tab a bunch
3. See error

**Expected behavior**

should have some indication the buttons are selected

**Screenshots**

![2019-07-18 16 28 22](https://user-images.githubusercontent.com/5615930/61498824-b8ea2380-a979-11e9-835c-f4f48a0e0be1.gif)

(gif is from arcade because I was testing tutorials there, but same applies to microbit)
